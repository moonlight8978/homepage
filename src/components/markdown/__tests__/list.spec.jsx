import React from 'react'
import { render } from '@testing-library/react'

import List from '../list'

test('renders unordered list', () => {
  const children = (
    <>
      <li>Item 1</li>
      <li>Item 2</li>
    </>
  )
  const { container } = render(<List ordered={false}>{children}</List>)
  expect(container.firstChild).toMatchSnapshot()
})

test('renders ordered list', () => {
  const children = (
    <>
      <li>Item 1</li>
      <li>Item 2</li>
    </>
  )
  const { container } = render(<List ordered>{children}</List>)
  expect(container.firstChild).toMatchSnapshot()
})
