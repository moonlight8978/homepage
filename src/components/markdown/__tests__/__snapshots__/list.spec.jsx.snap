// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ordered list 1`] = `
<ol
  class="list"
>
  <li>
    Item 1
  </li>
  <li>
    Item 2
  </li>
</ol>
`;

exports[`renders unordered list 1`] = `
<ul
  class="list"
>
  <li>
    Item 1
  </li>
  <li>
    Item 2
  </li>
</ul>
`;
